### PR TITLE
Added Order field to WebUrl.

### PR DESCRIPTION
**Pull request by PatrykSzwer**
For https://github.com/StarcounterApps/Website/issues/195

### CODE REVIEW

File: Ring6/Web/WebUrl.cs [hunk 1]
```diff
@@ -10,6 +10,7 @@ public class WebUrl : Something
         public bool IsFinal { get; set; }
         public bool Disabled { get; set; }
         public bool IsRegex { get; set; }
+        public int Order { get; set; }
```
Comment (un-tone) [inline Ring6/Web/WebUrl.cs]: We will have to escape this property in SQL, won't it? Let's rename it to `OrderNumber` or something similar

Comment (PatrykSzwer) [reply Ring6/Web/WebUrl.cs]: I wasn't sure about this also. Actually in WebMap class there is property called SortNumber. Maybe We should stick with one name and be consistent with this, or should I just left "OrderNumber"?

Comment (un-tone) [reply Ring6/Web/WebUrl.cs]: Let's name it `SortNumber` in this case

Comment (PatrykSzwer) [reply Ring6/Web/WebUrl.cs]: Check It now, should be ok.

**Pull request merged**